src/main/resources/META-INF/plugin.xml

```diff
@@ -22,6 +22,13 @@
                 id="network.radicle.jetbrains.radiclejetbrainsplugin.config.RadicleSettingsView"
                 groupWeight="-400"
                 displayName="Radicle"/>
+
+        <projectConfigurable
```

Comment:
After our discussion, I understand this is made project(-only)-configurable so that we can correctly test its behavior. We should experiment a bit whether it's possible to capture application-level notifications in our tests.